packages/dds/tree/src/test/shared-tree/treeNodeApi.spec.ts

```diff
@@ -216,29 +220,53 @@ describe("treeApi", () => {
 			});
 
 			// TODO: Either enable when afterBatch is implemented, or delete if no longer relevant
-			it.skip("emits change events", () => {
+			it("emits change events", () => {
```

Comment:
Remove the TODO above this?

Reply:
oh, these still fail, gotta put the skips back